source/jormungandr/jormungandr/interfaces/v1/serializer/pt.py

```diff
@@ -148,13 +148,13 @@ class ValidityPatternSerializer(PbNestedSerializer):
 
 
 class WeekPatternSerializer(PbNestedSerializer):
-    monday = BoolField()
-    tuesday = BoolField()
-    wednesday = BoolField()
-    thursday = BoolField()
-    friday = BoolField()
-    saturday = BoolField()
-    sunday = BoolField()
+    monday = BoolField(display_none=True)
```

Comment:
looks weird that you need this... supposed to be managed lower in our version of serpy. Are you sure you're up to date with pip?